### PR TITLE
Remove pandas index from glue job output

### PR DESCRIPTION
**Pull request by wrridgeway**

### CODE REVIEW

**Comment by wrridgeway** [comment]
This PR can chill until we're done mailing the South.

**Review by dfsnow — APPROVED** [review]
Let's merge this after the other sales view updates @wrridgeway. @ccao-jardine your Tableau views will likely break once this is merged :/

**Pull request merged**
